docs/search-filters.md

```diff
@@ -3,28 +3,33 @@
 The search experience can be improved by adding search filters to narrow down the results. The
 filters rely on listing's indexed data.
 
+* [Filter types](#filter-types)
+* [Adding a new search filter](#adding-a-new-search-filter)
+  * [Common changes](#common-changes)
+  * [Desktop filters](#desktop-filters)
+  * [Desktop filters panel](#desktop-filters-panel)
+  * [Mobile filters](#mobile-filters)
+* [Creating your own filter types](#creating-your-own-filter-types)
+
 ## Filter types
 
-The Flex template for web has three different filter types: _price filter_, _select single_ and
-_select multiple_. The _price filter_ is for the specific case of filtering listings with a price
-range.
+The Flex template for web has different filter types: _price filter_, _date range filter_, _select
+single_ and _select multiple_. The _price filter_ is for the specific case of filtering listings
+with a price range.
 
 > NOTE: price filter should be configured from `src/marketplace-custom-config.js`. Current maximum
 > value for the range is set to 1000 (USD/EUR).
 
-Other two filter types can be used with extended data. The _select single_ one can be used to filter
-out search result with only one value per search parameter. The _select multiple_ filters on the
-other hand can take multiple values for a single search parameter. These two filter types for
-extended data are implemented with four different components, a standard and a plain one:
+Filters _select single_ and _select multiple_ can be used with extended data. The _select single_
+one can be used to filter out search result with only one value per search parameter. The _select
+multiple_ filters on the other hand can take multiple values for a single search parameter. These
+two filter types for extended data are implemented with two different components:
 
-* Select single filter: `SelectSingleFilter` and `SelectSingleFilterPlain`
-* Select multiple filter: `SelectMultipleFilter` and `SelectMultipleFilterPlain`
+* Select single filter: `SelectSingleFilter`
+* Select multiple filter: `SelectMultipleFilter`
 
-The `SelectSingleFilter` and `SelectMultipleFilter` components are rendered as standard dropdowns in
-the search view. The plain filter components `SelectSingleFilterPlain` and
-`SelectMultipleFilterPlain` are used with `SearchFiltersMobile` and `SearchFiltersPanel` to provider
-filters in mobile view or to open filters in a distinct panel in order to fit more filters to the
-desktop search view.
+The `SelectSingleFilter` and `SelectMultipleFilter` components are rendered as dropdowns in the
+search view. They are used in `SearchFilters`, `SearchFiltersMobile`, `SearchFiltersPanel`.
```

Comment:
Now we're losing track a bit about what components are drop downs and which are used in the panel and mobile.

Maybe we could remove this paragraph here and mention this again in the end where we have FilterPopup and FilterPlain documented.